src/modules/search.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable import/prefer-default-export, object-curly-newline */
```

Comment:
we can remove the disabling of `import/prefer-default-export` here

Reply:
Nice catch!